pkg/apis/externaldns/types.go

```diff
@@ -96,6 +96,7 @@ type Config struct {
 	InfobloxSSLVerify                 bool
 	InfobloxView                      string
 	InfobloxMaxResults                int
+	InfobloxRecordsNameFilter         string
```

Comment:
I have a general question about the NameFilter.
How much impact does it have if you don't include the NameFilter?
What i was thinking: Is really necessary for multiple A records and should this pull be split into 2 changes:
* [feature] Add InfobloxRecordNameFilter for boosting query performance.
* [feature] Add support for multiple A records.

Benefits:
* Smaller pulls to review
* Easier to discuss.

Reply:
Its indeed needed to prevent to ddos infoblox. Not using the NameFilter will put limitations on the frequency of updating infoblox in case of large domains.